installer.tpl.sh

```diff
@@ -455,7 +457,7 @@ main ()
     echo "Setting up yum repository..."
     install_dnf
   elif ( [ ${os} = "debian" ] && [[ ${dist} =~ ^(jessie|stretch|buster|bullseye|bookworm)$ ]] ) ||
-       ( [ ${os} = "ubuntu" ] && [[ ${dist} =~ ^(trusty|xenial|bionic|cosmic|disco|eoan|focal|groovy|hirsute|impish|jammy)$ ]] ); then
+       ( [ ${os} = "ubuntu" ] && [[ ${dist} =~ ^(trusty|xenial|bionic|cosmic|disco|eoan|focal|groovy|hirsute|impish|jammy|noble)$ ]] ); then
```

Comment:
`static/installer.sh` also has a similar chunk of code.

Reply:
@Totktonada missed it, thanks. 

Fixed now. What should I do with centos:7 builds?

Reply:
CentOS 7 problem is irrelevant here, but if you're interested, you can replace the YUM repositories with the archived ones, similarly to how it is done in https://github.com/packpack/packpack-docker-images/pull/117.

Reply:
Filed #40.